[32m2023-06-02 18:26:47 +0200[0m - dagster - [34mDEBUG[0m - __ASSET_JOB - 29e7c5cc-d10a-43e0-a0fb-014ad7a1448c - 1171 - LOGS_CAPTURED - Started capturing logs in process (pid: 1171).
[32m2023-06-02 18:26:47 +0200[0m - dagster - [34mDEBUG[0m - __ASSET_JOB - 29e7c5cc-d10a-43e0-a0fb-014ad7a1448c - 1171 - move_from_raw_to_stg - STEP_START - Started execution of step "move_from_raw_to_stg".
[32m2023-06-02 18:26:47 +0200[0m - dagster - [34mERROR[0m - [31m__ASSET_JOB - 29e7c5cc-d10a-43e0-a0fb-014ad7a1448c - 1171 - move_from_raw_to_stg - STEP_FAILURE - Execution of step "move_from_raw_to_stg" failed.

dagster._core.errors.DagsterExecutionStepExecutionError: Error occurred while executing op "move_from_raw_to_stg"::

AttributeError: type object 'datetime.datetime' has no attribute 'datetime'

Stack Trace:
  File "/Users/ctac/Desktop/Data Engineer /Projects/realestate_scraping_project/env/lib/python3.11/site-packages/dagster/_core/execution/plan/utils.py", line 54, in op_execution_error_boundary
    yield
  File "/Users/ctac/Desktop/Data Engineer /Projects/realestate_scraping_project/env/lib/python3.11/site-packages/dagster/_utils/__init__.py", line 443, in iterate_with_context
    next_output = next(iterator)
                  ^^^^^^^^^^^^^^
  File "/Users/ctac/Desktop/Data Engineer /Projects/realestate_scraping_project/env/lib/python3.11/site-packages/dagster/_core/execution/plan/compute_generator.py", line 124, in _coerce_solid_compute_fn_to_iterator
    result = invoke_compute_fn(
             ^^^^^^^^^^^^^^^^^^
  File "/Users/ctac/Desktop/Data Engineer /Projects/realestate_scraping_project/env/lib/python3.11/site-packages/dagster/_core/execution/plan/compute_generator.py", line 118, in invoke_compute_fn
    return fn(context, **args_to_pass) if context_arg_provided else fn(**args_to_pass)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ctac/Desktop/Data Engineer /Projects/realestate_scraping_project/realestate-scraping/realestate_scraping/assets/core/realestate_scraping.py", line 314, in move_from_raw_to_stg
    context.resources.s3._move_object_between_buckets(raw_bucket, stg_bucket)
  File "/Users/ctac/Desktop/Data Engineer /Projects/realestate_scraping_project/realestate-scraping/realestate_scraping/__init__.py", line 57, in _move_object_between_buckets
    today = datetime.datetime.now().date().strftime("%y%m%d")
            ^^^^^^^^^^^^^^^^^
[0m
